### PR TITLE
Adding Mean.io

### PR DESCRIPTION
**Pull request by liorkesos**
Adding logo + mean regex (currently matching Mean 0.4.3 that contain the X-Powered-by ...

### CODE REVIEW

**Comment by DaAwesomeP** [comment]
Could you add that MEAN implies, `AngularJS`, `MongoDB`, `Expres`s, and `node.js`?

**Comment by liorkesos** [comment]
I updated the additional implies...

File: src/apps.json [hunk 1]
```diff
@@ -3378,6 +3378,21 @@
...
+				12
+			],
+    			"implies": [
+    			      "MongoDB",	
+			      "Express",
+			      "AngularJS",
+			      "node.js"
+			],
+			"headers": {
+				"X-Powered-CMS": "Mean.io"
```
Comment (gadcam) [inline src/apps.json]: You need to escape the dot like this <code>"X-Powered-CMS": "Mean\\\\.io"</code> ortherwise it will match any character.

**Comment by liorkesos** [comment]
Added the escape on the "."

**Comment by AliasIO** [comment]
Thanks!

**Pull request merged**